doc/references/ImplementationOfVHDL.rst

```diff
@@ -324,7 +324,7 @@ Linking with foreign object files
 ---------------------------------
 
 You may add additional files or options during the link using the
-*-Wl,* of `GHDL`, as described in :ref:`Elaboration_command`.
+*-Wl,* of `GHDL`, as described in ':ref:`Elaboration:command`'.
```

Comment:
My main goal here wasn't to add single quotes ``'`` but to fix the link. The [original](https://ghdl.readthedocs.io/en/latest/references/ImplementationOfVHDL.html#linking-with-foreign-object-files) was broken. The single quotes got added only because I was going off other examples within the rest of the documents.

Going back and looking at it now, I see that there were 16 links surrounded by single quotes ``'`` (see [Invoking GHDL](https://gritbub-ghdl.readthedocs.io/en/latest/using/InvokingGHDL.html) for 11 examples). Now that you mention it, I see that 22 other links are not surrounded by single quotes.

@1138-4EB I can do whichever you prefer.

Reply:
Then, I think that this is because some of the sections where last modified mostly by me, and others mostly by @Paebbels. We should agree on the format.

@Paebbels, @tgingold, @gritbub, which one do you prefer?

Reply:
I prefer no quotes around links. Links should have special formatting / highlighting, so not quotes are required to mark them as special text in the document.

Reply:
Done [here](https://github.com/ghdl/ghdl/pull/591/commits/7538ef0dbe9369294d5ca4a4fb9b14c87d893184)